.github/workflows/pr-check.yaml

```diff
@@ -0,0 +1,11 @@
+name: pr-check
+
+on:
+  pull_request:
+    branches: ['*']
+
+jobs:
+  test:
+    uses: ./.github/workflows/test.yaml
+    with:
+      environment: dev
```

Comment:
should this be ${{ inputs.environment }} instead of dev?

Reply:
no, this will trigger a run from a commit in any branch however the only valid inputs for environment are dev, stage and prod.